nixos/tests/systemd-networkd-vrf.nix

```diff
@@ -16,7 +16,8 @@ import ./make-test-python.nix ({ pkgs, lib, ... }: let
         linkConfig.RequiredForOnline = "no";
         networkConfig = {
           Address = "192.168.${toString vlan}.${toString id}/24";
-          IPForward = "yes";
+          IPv4Forwarding = "yes";
+          IPv6Forwarding = "yes";
```

Comment:
Unrelated to this PR: `"yes"` and `"no"` in this file would be better as `true` and `false`, as that's more Nix-ish. But it doesn't matter.